cli_tools/common/utils/param/fixer.go

```diff
@@ -0,0 +1,77 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package param
+
+import (
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/domain"
+)
+
+// Fixer standardizes user input, and determines omitted values.
+type Fixer interface {
```

Comment:
I rarely see the word "Fixer" used for this purpose. Not sure whether there is a better word.

Reply:
I struggled as well. Consumers outside this package see `param.Fixer`, which seems reasonable.

Reply:
+1 on this. Fixed implies something is broken (like parameters having incorrect values, when actually parameters don't have values set at all).

Reply:
`param.Fixer` was the best I could think of. You have some ideas on alternatives?

Reply:
Populator?